models/hook/hook.go

```diff
@@ -0,0 +1,115 @@
+package hook
+
+import (
+	"../../config"
+	"../../modules/hub"
+	"../../modules/logging"
+	"../db"
+	"../list"
+	"../project"
+	"../task"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/go-github/github"
+)
+
+func IssuesEvent(repositoryID int64, body github.IssuesEvent) error {
```

Comment:
__[golint]__
* [golint] exported function IssuesEvent should have comment or be unexported